AtDigitization/AtDigiLinkDef.h

```diff
@@ -13,5 +13,6 @@
 #pragma link C++ class AtSimulatedLine + ;
 #pragma link C++ class AtTrigger + ;
 #pragma link C++ class AtTriggerTask + ;
+#pragma link C++ class AtVectorResponse - ;
```

Comment:
This should probably have a "!" also.

Reply:
where should the "!" be? What is it?

Reply:
`#pragma link C++ class AtVectorResponse - !;`

It disables the generation of a `<<` operator.